test/config-next/ca.json

```diff
@@ -39,7 +39,6 @@
     "backdate": "1h",
     "lifespanOCSP": "96h",
     "maxNames": 100,
-    "doNotForceCN": true,
```

Comment:
Why are we removing this?

Reply:
It produces lint errors, which are reasonably correct. With this flag (which isn't on in prod), we leave out the CN and add a serialNumber, but that's not really valid under BRs.

Reply:
Ah yeah I remember the argument about why `serialNumber` ended up not being ok. Since we can't switch this on until there is changes in the BR (and probably in that case we'd actually want something else instead of `serialNumber`) can we just remove this code in a follow-up change?

Reply:
Yep, that's that plan.